### PR TITLE
Add jetstream request

### PR DESCRIPTION
**Pull request by Jarema**

### CODE REVIEW

File: Sources/JetStream/context.swift [hunk 1]
```diff
@@ -145,3 +165,11 @@ struct Ack: Codable {
         duplicate = try container.decodeIfPresent(Bool.self, forKey: .duplicate) ?? false
     }
 }
+
+/// contains info about the `JetStream` usage from the current account.
+public struct AccountInfo: Codable {
```
Comment (piotrpio) [inline Sources/JetStream/context.swift:170]: We can do it in a separate PR, but this is not a full account info response

**Review by piotrpio — APPROVED** [review]
LGTM!

File: Sources/JetStream/context.swift (hunk 1 above)
Comment (Jarema) [reply Sources/JetStream/context.swift:170]: Yeah, I just wanted something that I can test against.

Let's make work on it when we will add the AccountInfo PR.
